.gitignore

```diff
@@ -49,6 +49,7 @@ local.properties
 node_modules/
 npm-debug.log
 yarn-error.log
+package-lock.json
```

Comment:
You should really use yarn! We all do, so there is no lockfile for npm checked in, only `yarn.lock`. So maybe I should say, you have to use yarn ;)

Reply:
Okay, I'll use yarn!